### DISCUSSION TITLE
Copy entry from one database, paste into another

### DISCUSSION THREAD

**Discussion by flagellatedcucumber** [keepassxreboot/keepassxc#6317] [Q&A] [answered]
I need to split one database into a few smaller ones.

In Keepass, I know how to copy an entry from one database, paste it into the other database.

How do I do this in KeepassXC v2.4.3 Linux edition?

**Comment by droidmonkey** [comment, marked as answer]
Drag and drop. You hold CTRL after dragging to do a copy operation. Alternatively you can just create a copy of the database file itself and delete the groups and entries you don't want. Make sure you reset the database UUID in the browser settings if you do this.

**Comment by flagellatedcucumber** [comment]
Thanks - that works.

Feature request: can we have a keyboard/menu driven means of doing this, instead of relying upon a mouse?

**Reply by droidmonkey** [reply]
This has been requested over on the issues board

**Comment by luka-jova** [comment]
Hi! When doing so, you need to drop into the target directory listed under Group Panel, rather than into the entry list.

Also, copying is achieved by pressing CTRL before drag-and-drop, rather than during the drag.

At least it works like that on KeepassXC version 2.7.10 for Linux.

**Reply by droidmonkey** [reply]
Every platform is different in terms of the Ctrl behavior
